### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
# Hi there 😄 
> In this PR, you will find:

- [x] I created the following components for the react app:

            calculator.js
            numerickeys.js
            DisplayScreen.js
            operatorkeys.js
     
- [x] I styled the components to replicate the same UI wireframe in project requirements
- [x] I added. committed, and pushed changes

## Kindly Review

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+import './style.css';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>
+    <div className="calculator-container">
+      <Calculator />
```
Comment (gemmen29) [inline src/index.js]: - [ ] Kindly import the `calculator` component inside the main component `app.js` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Once%20the%20Calculator%20is%20ready%2C%20import%20it%20and%20use%20it%20in%20your%20main%20component%20(App.js).) and restore the import of `app.js` inside the `index.js` and make sure `app.js` is class component.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @PromzzyKoncepts, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (peterrobert) [reply src/index.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @PromzzyKoncepts, 

### HIGHLIGHTS

- Great work making the changes that was requested 👍🏿 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**